src/utils/dispute-utils.js

```diff
@@ -464,6 +540,13 @@ export function getRoundFees(round, courtConfig) {
     .mul(round.jurorsNumber)
 }
 
+/**
+ *
+ * @param {BigNum} minActiveBalance The minimum active balance required to become an active juror
+ * @param {BigNum} penaltyPct Permyriad of min active tokens balance to be locked to each drafted juror
```

Comment:
> Permyriad of min active tokens balance

I really don't know what this means haha, it looks like "pyramid" to me but that can't be right!

Reply:
Haha i wasn't familiar with the word either until i looked at the contracts :D 
For what i understand it means `a % of..` 

Will change the term :)

Reply:
Looool